packages/headless/src/database/mongodb/branch/index.ts

```diff
@@ -8,7 +8,14 @@ import { BranchModel } from './branch.model';
 @Injectable()
 export class BranchDatabase implements IBranchDatabase {
     async createBranch(branch: CreateBranchRequest): Promise<Branch | null>{
-        return await BranchModel.create(branch);
+      let newBranch;
+      try {
+        newBranch = await BranchModel.create(branch);
```

Comment:
you may return just one line.
like: 
` return await BranchModel.create(branch);`